internal/impl/kafka/enterprise/integration_test.go

```diff
@@ -376,6 +395,19 @@ func TestSchemaRegistryIntegration(t *testing.T) {
 				resp, err = http.DefaultClient.Do(req)
 				require.NoError(t, err)
 				require.NoError(t, resp.Body.Close())
+				require.Equal(t, http.StatusOK, resp.StatusCode)
+			}
+
+			if test.schemaWithReference != "" {
+				body, err := json.Marshal(payload{Schema: test.schemaWithReference, References: []map[string]any{{"name": "foo", "subject": subject, "version": 1}}})
```

Comment:
A bit janky, but I can clean it up to use the name from `t.schema` if need be...